src/index.html

```diff
@@ -4,9 +4,412 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo" href="/">
+        <img src="images/logo.png" alt="MOYO logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__ul">
+          <li class="nav__item">
+            <a
+             class="nav__item-link is-active"
+             href="/apple"
+            >
+             Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="/samsung">Samsung</a>
```

Comment:
This string looks too long and it's harder to read it
Let's move inside value on the new line 
Check it also for next links

```suggestion
            <a class="nav__item-link" href="/samsung">
              Samsung
            </a>
```